src/App.tsx

```diff
@@ -1,223 +1,227 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React, { useContext, useEffect, useRef } from 'react';
+import React, {
+  useCallback,
+  useContext,
+  useEffect,
+  useMemo,
+  useRef,
+  useState,
+} from 'react';
+import { getFilterTodos } from './api/helper';
+import {
+  createTodo,
+  deleteTodoById,
+  getTodos,
+  updateTodoOnServer,
+} from './api/todos';
 import { AuthContext } from './components/Auth/AuthContext';
+import { ErrorNotification }
+  from './components/ErrorNotification/ErrorNotification';
+import { Footer } from './components/Footer/Footer';
+import { Header } from './components/Header/Header';
+import { TodoList } from './components/TodoList/TodoList';
+import { FilterType } from './types/FilterType';
+import { Todo } from './types/Todo';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [isError, setIsError] = useState<boolean>(false);
+  const [errorMessage, setErrorMessage] = useState<string>('');
+  const [filterStatus, setFilterStatus] = useState<string>(FilterType.ALL);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [isAdding, setIsAdding] = useState(false);
+
+  const [updatingTodosIds, setUpdatingTodoIds] = useState<number[]>([]);
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
No need for this linter disabler anymore
```suggestion
```

Reply:
Yeap, I agree with @oleh-yablunovskyi